src/components/VideoPlayer.vue

```diff
@@ -613,7 +613,7 @@ export default {
                         // Set the audio language
                         const prefLang = this.getPreferenceString("hl", "en").substr(0, 2);
                         var lang = "en";
-                        for (var l in player.getAudioLanguages()) {
+                        for (var l of player.getAudioLanguages()) {
```

Comment:
We could actually simplify this more, i.e.
```suggestion
          const lang = player.getAudioLanguages().includes(prefLang) ? prefLang : "en";
```
but that can be done in a different commit (I'll push that shortly after merging your PR).